pkg/costmodel/assets_handler.go

```diff
@@ -0,0 +1,34 @@
+package costmodel
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/opencost/opencost/pkg/env"
+	"github.com/opencost/opencost/pkg/kubecost"
+	"github.com/opencost/opencost/pkg/util/httputil"
+)
+
+// ComputeAllocationHandler returns the assets from the CostModel.
+func (a *Accesses) ComputeAssetsHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
```

Comment:
Should we just name it `handlers.go `? Are we going to have more handlers off costmodel.

Go lang file names are not delimited by underscore unless test files so going to be assetsHandlers.go if its retained.

Reply:
I think `handlers` is good!